blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubIssue.java

```diff
@@ -65,7 +62,7 @@ public String getURL() {
                 return null;
             }
             GitHubSCMSource gitHubSource = (GitHubSCMSource)source;
-            String apiUri = MoreObjects.firstNonNull(gitHubSource.getApiUri(), GitHubServerConfig.GITHUB_URL);
+            String apiUri = Objects.firstNonNull(gitHubSource.getApiUri(), GitHubServerConfig.GITHUB_URL);
```

Comment:
couldn't this just be trivial code vs a guava import?

Reply:
```
String apiUri = (gitHubSource.getApiUri() == null) ? gitHubSource.getApiUri() : GitHubServerConfig.GITHUB_URL;
```

Reply:
This sounds fine to me 👍. However there are other instances where we are using guava, those uses can surprise us. Either we should somehow enforce so that development time we see version of guava coming from core, or at least compile time it generates error if there are usage from later version.

Or stop using guava or shade it or start using jdk8 for some of its functionalities are over there or write such utilities ourselves.